subsys/net/lib/nrf_provisioning/src/nrf_provisioning.c

```diff
@@ -562,6 +563,26 @@ int nrf_provisioning_req(void)
 			dm.cb(NRF_PROVISIONING_EVENT_STOP, dm.user_data);
 		}
 
+		if (ret == -EACCES) {
+			if (IS_ENABLED(CONFIG_NRF_PROVISIONING_PRINT_ATTESTATION_TOKEN)) {
+				struct nrf_attestation_token token = { 0 };
+				int err;
+
+				err = modem_attest_token_get(&token);
+				if (err) {
+					LOG_ERR("Failed to get token, err %d", err);
+				} else {
+					printk("\nAttestation token "
```

Comment:
Should this be printk, or LOG_INFO/LOG_WRN?

Reply:
I specifically chose printk to make the log of attestation token more obvious and readable. So much of what gets logged is not actionable by the user -- I want this information to stand out, and to not be suppressed by log level.

Reply:
I'd argue this is a good use-case for LOG_ERR or LOG_WRN, then, since those will not usually be suppressed -- That way it stays congruent with the logging subsystem

Reply:
Nope.